src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -239,6 +239,26 @@ public void ExtensionWarningEvent(string hubName, string functionName, string in
             }
         }
 
+        public void ProcessingOutOfProcPayload(
+            string functionName,
+            string taskHub,
+            string instanceId,
+            string details)
+        {
+            EtwEventSource.Instance.ProcessingOutOfProcPayload(
+                functionName,
+                taskHub,
+                LocalAppName,
+                LocalSlotName,
+                instanceId,
+                details,
+                ExtensionVersion);
+
+            this.logger.LogInformation(
+                "{instanceId}: Function '{functionName} ({functionType})' is being replayed. Details: {details}. : {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
```

Comment:
I would change this message a bit. Instead of  "XXX is being replayed", change it to something like "XXX returned the following orchestrator actions: "

Reply:
I went for "returned the following OOProc orchestration state" because we're not just logging the actions, that would be more expensive that simply outputting the orchestration state, which only contains ~3 more fields in addition to the actions array